src/ui/components/SecondaryToolbox/ReduxDevTools.tsx

```diff
@@ -37,9 +38,12 @@ export const ReduxDevToolsPanel = () => {
   const reduxAnnotations: ReduxActionAnnotation[] = useMemo(() => {
     const annotations = annotationsStatus === "resolved" ? parsedAnnotations : [];
     return annotations.filter(
-      annotation => focusWindow && isPointInRegion(annotation.point, focusWindow)
+      annotation =>
```

Comment:
:hand: Probably need to check if there even is a non-empty `searchValue` before we do that comparison

Reply:
I don't think that's required with `String#match` as it treats the second string as a regex, but on second thought `String#includes` is probably better in this case